HISTORY.rst

```diff
@@ -45,7 +45,7 @@ History
 
 * Bug fix, error in manifest of files to include in python package (#120)
 
-Unreleased
```

Comment:
Should the Unreleased heading remain, so it doesn't have to be remembered on the next edit?